test/tests/functional/pbs_qstat_formats.py

```diff
@@ -580,6 +580,20 @@ def test_qstat_json_valid_job_special_env(self):
         """
         os.environ["DOUBLEQUOTES"] = 'hi"ha'
         os.environ["REVERSESOLIDUS"] = r'hi\ha'
+        os.environ["MYVAR"] = """\'\"asads\"\'"""
+        os.environ["MYVAR2"] = """/home/pbstest01/Mo\\'"""
+        os.environ["FOO"] = """005"""
+        os.environ["MYVAR1"] = """\'"""
+        os.environ["MYVAR2"] = """\\'"""
+        os.environ["MYVAR3"] = """\\\\'"""
```

Comment:
Can you test `"""\\\'"""`?
That's three backslashes followed by a single quote.

Reply:
Sure.